packages/cli/test/setup.js

```diff
@@ -4,13 +4,11 @@ import { ZWeb3, Contracts } from 'zos-lib'
 import Dependency from '../src/models/dependency/Dependency'
 import ZosPackageFile from '../src/models/files/ZosPackageFile'
 import ZosNetworkFile from '../src/models/files/ZosNetworkFile'
-const prompt = require('../src/prompts/prompt');
 
 useTestZosPackageFile()
 doNotInstallStdlib()
 ZWeb3.initialize(web3.currentProvider)
 setArtifactDefaults()
-prompt.DEFAULT_INTERACTIVE_STATUS = false
```

Comment:
Isn't this required to prevent the tests in `test/commands` from being interactive?

Reply:
I think no. Two reason:
* DEFAULT_INTERACTIVE_STATUS was never used because commander.js always set `interactive` flag regardless of will you provide it or not
* I think all our commands tests doesn't ever trigger interactivity.